ground/src/main/java/com/google/android/ground/persistence/remote/firestore/schema/SubmissionCollectionReference.kt

```diff
@@ -36,7 +36,7 @@ class SubmissionCollectionReference internal constructor(ref: CollectionReferenc
     fun submissionsByLocationOfInterestId(
         locationOfInterest: LocationOfInterest
     ): @Cold Single<ImmutableList<ValueOrError<Submission?>>> {
-        return RxFirestore.getCollection(byLoiId(locationOfInterest.id))
+        return RxFirestore.getCollection(byLoiId(locationOfInterest.id ?: ""))
```

Comment:
Isn't this incorrect? Loi ID should never be empty.

Reply:
+1. Though there are many cases where we could benefit from making model attributes non-null; we have a TODO in our planning doc to fix so feel free to address in a separate PR.

Reply:
sg, let's tackle this separately.